dotnet/src/Connectors/Connectors.Memory.CosmosDB/CosmosDBMemoryRecord.cs

```diff
@@ -1,18 +1,16 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
 using System;
-using System.Collections.Generic;
-using Microsoft.SemanticKernel.Memory;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
 
 namespace Microsoft.SemanticKernel.Connectors.Memory.Cosmos;
 
 /// <summary>
-/// A Cosmos memory record.
```

Comment:
I switched from "CosmosDB" to "Cosmos" to follow the Azure naming conventions for Cosmos DB.

Reply:
renamed CosmosDB -> Cosmos